php/chubbyphp-roadrunner/config.yaml

```diff
@@ -0,0 +1,8 @@
+framework:
+  github: chubbyphp/chubbyphp-framework
+  version: 2.8
+
+before_command:
+  - vendor/bin/rr get-binary
```

Comment:
@dominikzogg How can the command be different from `spiral` ?
https://github.com/the-benchmarker/web-frameworks/blob/cb5bd67bddd75db632828bb72f79d90d5347a013/php/spiral/config.yaml#L9

BTW, this is working, I have just curiosity in mind

Reply:
@waghanza spiral got some wrapper code, i use the plain roadrunner as suggested within the roadrunner documentation

Reply:
:ok_hand: